### DISCUSSION TITLE
I am writing posts/tutorials on MMDetection

### DISCUSSION THREAD

**Discussion by sovit-123** [open-mmlab/mmdetection#8132] [Ideas]
Hello, my name is Sovit. I am the author of the deep learning blog DebuggerCafe (https://debuggercafe.com/).
Currently, I am preparing four definitive blog posts for MMDetection (names may change later on).
1. A definitive guide to installing MMDetection on both Ubuntu and Windows for both RTX and GTX GPUs.
2. A detailed guide to image and video inference using MMDetection. 
3. Custom object detection on a small benchmark dataset using MMDetection.
4. A detailed guide to custom object detection using MMDetection.

I wanted to know if they turn out well enough (the official contributors of the repository will surely verify that first), is it possible to add the blog post links here for others to refer to? I think they will be pretty helpful.

**Comment by ZwwWayne** [comment]
Hi @sovit-123 ,
Yeah! We are glad to add those blog post links here for others to refer to.
Looking forward to your new blogs!

**Comment by sovit-123** [comment]
@ZwwWayne Thanks a lot. I am currently writing the posts. I will again post here once all of them are published.

**Comment by sovit-123** [comment]
Hello @ZwwWayne 
I have written 4 posts on MMDetection that I told you about earlier. I am linking them here. If you or anyone else can take a look at them to verify whether they can be added as tutorial resources, I will be really glad.
1. [Install MMDetection on Ubuntu and Windows for RTX and GTX GPUs ](https://debuggercafe.com/install-mmdetection-on-ubuntu-and-windows-for-rtx-and-gtx-gpus/)
2. [Image and Video Inference using MMDetection](https://debuggercafe.com/image-and-video-inference-using-mmdetection/)
3. [Getting Started with MMDetection Training for Object Detection](https://debuggercafe.com/getting-started-with-mmdetection-training-for-object-detection/)
4. [Custom Dataset Training using MMDetection](https://debuggercafe.com/custom-dataset-training-using-mmdetection/)

Thanks a lot!
